br/pkg/conn/conn.go

```diff
@@ -219,14 +219,6 @@ func NewMgr(
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		// we must check tidb(tikv version) any time after concurrent ddl feature implemented in v6.2.
-		// we will keep this check until 7.0, which allow the breaking changes.
```

Comment:
maybe it means BR can run on a old version cluster. @Leavrth can you take a look?